src/Microsoft.Azure.WebJobs.Host/JobHostConfiguration.cs

```diff
@@ -65,15 +66,15 @@ public JobHostConfiguration(string dashboardAndStorageConnectionString)
 
             AddService<IExtensionRegistry>(extensions);
             AddService<StorageClientFactory>(new StorageClientFactory());
-            AddService<INameResolver>(new DefaultNameResolver());
+            AddService<INameResolver>(new AppSettingOrEnvironmentNameResolver());
```

Comment:
So this is the default, but if someone sets JobHostConfiguration.NameResolver themselves, they'd override this, right?